src/gax/src/lib.rs

```diff
@@ -79,3 +79,4 @@ pub mod http_client;
 pub mod backoff_policy;
 pub mod options;
 pub mod retry_policy;
+pub mod retry_throttler;
```

Comment:
nit: I wonder if we should have a retry mod to group all of these cool things you have been building.

Reply:
🤔  maybe, I think this is the last one, but we can reconsider after I merge the complete loop.